app/src/test/java/io/gnosis/safe/ui/settings/owner/OwnerSelectionViewModelTest.kt

```diff
@@ -61,7 +61,7 @@ class OwnerSelectionViewModelTest {
             ownerCredentialsVault.storeCredentials(any())
             settingsHandler.showOwnerBanner = false
             settingsHandler.showOwnerScreen = false
-            tracker.logKeyImported()
+            tracker.logKeyImported(any())
```

Comment:
Since we know `logKeyImported()` should be passed `true` here, why not very it is?